packages/next/test/fixtures/00-app-dir-ppr-full/next.config.js

```diff
@@ -0,0 +1,14 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  experimental: {
+    ppr: true,
+  },
+  eslint: {
+    // Warning: This allows production builds to successfully complete even if
+    // your project has ESLint errors.
+    ignoreDuringBuilds: true,
+  },
```

Comment:
Can we remove eslint from the fixture? or is it baked into Next.js?

Reply:
It's baked in to Next.js's build process.